### PR TITLE
Added Doris Lee

### PR DESCRIPTION
**Pull request by kngoledge**

### CODE REVIEW

**Comment by prwatch** [comment]
DEMO: http://kngoledge.github.io/gci15.fossasia.org

File: _data/mentors.yml [hunk 1]
```diff
@@ -194,4 +194,13 @@
   github: hossainalikram
   image: hossain.jpg
   twitter: Hossainalikram
+  bio: 
+  
+  - name: Doris Lee
```
Comment (jig08) [inline _data/mentors.yml]: Hey @kngoledge 
Follow the pattern. There should not be any space before `- name`

Comment (mananwason) [inline _data/mentors.yml:199]: @kngoledge why is there an indentation here? Please remove this.

Comment (jigyasa-grover) [inline _data/mentors.yml:199]: @kngoledge Follow the pattern. There should not be any space before `- name` 

EDIT: @mananwason Do we revert this PR ?

Comment (roonyh) [inline _data/mentors.yml:199]: @mananwason @jig08 No need to revert the PR. This is not a syntax error right? This could be fixed in another commit.

Comment (mananwason) [inline _data/mentors.yml:199]: @jig08 @roonyh I agree. No point reverting the PR now, there will be extra commits for reverting as well. We can fix it in future commits.

Comment (jigyasa-grover) [inline _data/mentors.yml:199]: @roonyh @mananwason :+1:

Comment (mananwason) [inline _data/mentors.yml:199]: @roonyh @jig08 just checked the website before this commit. One of the mentors hossain's record has vanished. We need to fix that.

Comment (jigyasa-grover) [inline _data/mentors.yml:199]: In my opinion that's due to the faulty indentation ....

Comment (mananwason) [inline _data/mentors.yml:199]: Yeah it is obviously due to faulty indentation but we need to fix that either by reverting or in future commits.

Comment (jigyasa-grover) [inline _data/mentors.yml:199]: Fixed via PR #351

**Pull request merged**